src/components/Card.js

```diff
@@ -48,7 +66,50 @@ export default class Card {
     const imageElement = this._cardElement.querySelector(".card__image");
     imageElement.src = this.link;
     imageElement.alt = this.name;
+    this._cardElement.id = this.imageID;
+
     this._cardElement.querySelector(".card__caption").textContent = this.name;
+    if (this.myID !== this.ownerId) {
+      this._cardElement.querySelector(".card__delete-button").remove();
+    }
+
+    if (this.likes.length > 0) {
+      this.likes.forEach((user) => {
+        if (user._id === this.myID) {
+          this._cardElement
+            .querySelector(".card__like-button")
+            .classList.toggle("card__like-button_active");
+        }
+      });
+
+      this._cardElement.querySelector(".card__like-count").textContent =
+        this.likes.length;
+    }
+  }
+
+  isLiked() {
+    // return true if card is liked by user, otherwise false
+    // loop through list of likes
+    // if any like._id matches this.myID, return true
+
+    let likeUser = null;
+
+    for (let i = 0; i < this.likes.length; i++) {
```

Comment:
The code can be optimized. you can use `Array.some` method  `this._likes.some((like) => like._id === this._userId)`